recipes/f77_zmq/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "f77_zmq" %}
+{% set version = "4.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zeromq/f77_zmq/releases/download/v4.3.2_autotools/f77-zmq-4.3.2.tar.gz
+  sha256: e027bf1017ca6b47ce4a3481ae5b255db8ea86294bc32f1207bd90dbdece7671
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - make
+  host:
+    - zeromq
+  run:
+    - zeromq
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libf77zmq.so     # [linux]
+    - test -f ${PREFIX}/lib/libf77zmq.dylib  # [osx]
```

Comment:
Test for the existence of a header?

Reply:
No, there is no additional header file.

Reply:
This is a Fortran77 binding. Interfaces and modules were introduced in Fortran90. So there are no header files.

Reply:
OK. If headers are not needed, then don't ship them. The build artifacts contain two `*.h` files.

Reply:
Oh yes, you are right! My mistake: these two files are indeed header files that are needed by the user. They are old-style definitions that the user need to include. Sorry about that.
I now added the test for the presence of these two files.
Thanks for having looked carefully :-)